frontend/src/features/admin-form/common/components/CollaboratorModal/CollaboratorListScreen/CollaboratorRow.tsx

```diff
@@ -13,7 +13,7 @@ export const CollaboratorText: FC = ({ children }) => {
     <Text
       textStyle={{ base: 'subhead-1', md: 'body-2' }}
       color={{ base: 'secondary.700', md: 'secondary.500' }}
-      isTruncated
+      noOfLines={{ base: 0, md: 1 }}
```

Comment:
If this is truncated, is there any other way of seeing the full emails of existing collaborators?

Reply:
good point. added a `title`; hovering the row would now display the full email.